### ISSUE TITLE
Add command line arguments

### ISSUE THREAD

**Issue by alpgarcia** [alpgarcia/armin#2]
Arguments:
 - [ ] Out file name
 - [ ] Select evidence extractor (bm25, ontobio, etc.)
 - [ ] Training set path
 - [ ] Test set path
